data/xml/W19.xml

```diff
@@ -18152,9 +18152,9 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
       <author><first>Chenghua</first><last>Lin</last></author>
       <pages>124–129</pages>
       <abstract>A prominent strand of work in formal semantics investigates the ways in which human languages quantify over the elements of a set, as when we say “<i>All <i>A</i> are <i>B</i>
-        </i>”, “<i>All except two <i>A</i> are <i>B</i>
-        </i>”, “<i>Only a few of the <i>A</i> are <i>B</i>
-        </i>” and so on. Our aim is to build Natural Language Generation algorithms that mimic humans’ use of quantified expressions. To inform these algorithms, we conducted on a series of elicitation experiments in which human speakers were asked to perform a linguistic task that invites the use of quantified expressions. We discuss how these experiments were conducted and what corpora they gave rise to. We conduct an informal analysis of the corpora, and offer an initial assessment of the challenges that these corpora pose for Natural Language Generation. The dataset is available at: <url>https://github.com/a-quei/qtuna</url>.</abstract>
+ </i>”, “<i>All except two <i>A</i> are <i>B</i>
+ </i>”, “<i>Only a few of the <i>A</i> are <i>B</i>
+ </i>” and so on. Our aim is to build Natural Language Generation algorithms that mimic humans’ use of quantified expressions. To inform these algorithms, we conducted on a series of elicitation experiments in which human speakers were asked to perform a linguistic task that invites the use of quantified expressions. We discuss how these experiments were conducted and what corpora they gave rise to. We conduct an informal analysis of the corpora, and offer an initial assessment of the challenges that these corpora pose for Natural Language Generation. The dataset is available at: <url>https://github.com/a-quei/qtuna</url>.</abstract>
```

Comment:
What happened here? Not the only instance of this.

Reply:
I think it be would useful to be consistent with how lxml writes files, though I haven't checked this particular instance.

Reply:
In general we are pretty consistent, logic for which is captured in `indent()` in `anthology/utils.py`. Lots of the corrections here are removing inconsistencies (e.g., trailing spaces in abstracts). I think the issue here is that it's a bit of a complicated exception where the input was wrong. I'll correct it manually.